pom.xml

```diff
@@ -682,6 +682,31 @@ THE SOFTWARE.
       <id>release</id>
       <build>
         <plugins>
+          <plugin>
+            <artifactId>maven-deploy-plugin</artifactId>
+            <configuration>
+              <altDeploymentRepository>releases::${ env.MAVEN_REPOSITORY_URL}/maven-releases/</altDeploymentRepository>
+              <altSnapshotDeploymentRepository>releases-snapshot::${ env.MAVEN_REPOSITORY_URL }/maven-snapshots</altSnapshotDeploymentRepository>
+              <deployAtEnd>false</deployAtEnd>
+              <retryFailedDeploymentCount>3</retryFailedDeploymentCount>
+            </configuration>
+          </plugin>
+          <plugin>
+            <artifactId>maven-release-plugin</artifactId>
+            <configuration>
+              <!-- enable release profile during the release, create IPS package, and sign bits. -->
+                    <!-- Skip Test <arguments>-P release,sign -DskipTests=true</arguments>-->
+              <arguments>-P release,sign</arguments>
+              <!-- work around for a bug in javadoc plugin that causes the release to fail. see MRELEASE-271 -->
+              <preparationGoals>clean install</preparationGoals>
+                  <!-- Skipp Test <goals>-DskipTests -Danimal.sniffer.skip=false javadoc:javadoc deploy</goals>-->
+              <goals>-Danimal.sniffer.skip=false javadoc:javadoc deploy</goals>
+              <!--<goals>javadoc:javadoc deploy</goals>-->
+              <pushChanges>false</pushChanges>
```

Comment:
How I have come to loathe `maven-release-plugin`…

Reply:
I got to that point years ago and the feeling hasn't gotten any better.